src/main/java/org/embulk/output/elasticsearch/ElasticsearchHttpClient.java

```diff
@@ -265,11 +266,43 @@ private void deleteIndex(String indexName, PluginTask task, Jetty92RetryHelper r
         // curl -XDELETE localhost:9200/{index}
         // Success: {"acknowledged":true}
         if (isIndexExisting(indexName, task, retryHelper)) {
+            getSnapshotStatusUntilDone(task, retryHelper);
             sendRequest(indexName, HttpMethod.DELETE, task, retryHelper);
             log.info("Deleted Index [{}]", indexName);
         }
     }
 
+    private void getSnapshotStatusUntilDone(PluginTask task, Jetty92RetryHelper retryHelper)
```

Comment:
minor: "wait" is better name of this method. Because someone've taken snapshots. Instead, the plugin waits the snapshots completed. How about this?